static/app/views/starfish/components/tableCells/countCell.tsx

```diff
@@ -0,0 +1,18 @@
+import {NumberContainer} from 'sentry/utils/discover/styles';
+import {formatAbbreviatedNumber} from 'sentry/utils/formatters';
+
+type Props = {
+  count: number;
+  containerProps?: React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  >;
+};
+
+export default function DurationCell({count, containerProps}: Props) {
```

Comment:
Small copypasta error! Also, would be better as a named export
```suggestion
export function CountCell({count, containerProps}: Props) {
```

Reply:
Thanks for catching!